src/tools/mmaps_generator/PathGenerator.cpp

```diff
@@ -95,8 +95,7 @@ bool handleArgs(int argc, char** argv,
             param = argv[++i];
             if (!param)
                 return false;
-            threads = atoi(param);
-            printf("Using %i threads to extract mmaps\n", threads);
+            threads = static_cast<unsigned int>(std::max(0, atoi(param)));
```

Comment:
Preventing 0 as thread count will disable one "feature" of the program as it is.
See this:
https://github.com/Rochet2/TrinityCore/blob/f41c26761df4272fe8343fc61439f7cc06af04c9/src/tools/mmaps_generator/MapBuilder.cpp#L212